### DISCUSSION TITLE
ACF replacement?

### DISCUSSION THREAD

**Discussion by unclescooby** [affiliatetheme/affiliatetheme#9] [Ideas]
Hi,

Might I suggest replacing ACF pro with another plugin?

First, ACF was (and probably still is) creating duplicate metadata, which can add up on sites with lots of products with lots of meta fields. See the raw metadata links on [this page](https://www.billerickson.net/wordpress-metaboxes/). The only excuse for extra data would have to be that it makes the plugin more performant.

Second, cost. There an various plugins with same/similar features, that are totally free! For instance CMB2, pods, carbon fields. Not sure if any of them has all needed features, easy to integrate, and offers good performance. That would be for you to figure out ;) A good place to start might be [this comparison table](https://docs.google.com/spreadsheets/d/1mSqienVYxLopTFGLPK0lGCJst2knKzXDtLQRgwjeBN8/).

EDIT: obviously, not having to spend money up front on acf pro license may be good for affiliatetheme popularity, and for later spends on paid affiliatetheme add-ons.

**Comment by StrongLucky** [comment]
Hi,

thanks for your idea. We will have a look on your point. However, our focus will be more on new features in the near future, as ACF is working well.

The price for ACF is manageable in our opinion. Also consider that we have the coupon code with 30% for it: AFFILIATETHEME-G6LP23 (You can also find it in ourN FAQ: https://affiliatetheme.io/affiliatetheme-2-0-faq/)

I moved the topic to the discussions, as it is not a mistake ;)

Lucas
